src/components/CommentList/CommentList.jsx

```diff
@@ -1 +1,12 @@
-export const CommentList = () => <>Put the list here</>;
+import './CommentList.scss';
+import { CommentInfo } from '../CommentInfo/CommentInfo';
```

Comment:
We have `index.js`, so we don't need duplicated way to the file

```suggestion
import { CommentInfo } from '../CommentInfo';
```